add outline-only versions of the OSMC 'pointer' symbol

mueschel opened the pull request:
Adding three new symbols - line versions of the "pointer" symbols:   pointer_line, left_pointer_line, right_pointer_line.

As suggested in https://community.openstreetmap.org/t/brauche-hilfe-fur-wanderweg-symbol/122926

Commit:
add outline-only versions of the OSMC 'pointer' symbol

lonvia commented on wmt_shields/styles/osmc_symbol.py:
```diff
@@ -280,7 +280,31 @@ def paint_fg_left_pointer(self, ctx):
+        ctx.line_to(0.1, 0.1)
+        ctx.stroke()
+
+    def paint_fg_right_pointer_line(self, ctx):
```
We don't have a 'right_pointer', so I wouldn't want to introduce this here.

lonvia reviewed (changes requested):
Looks okay to me, except for the 'right_pointer_line'.

mueschel commented:
The right_pointer I already added a while ago:
https://github.com/waymarkedtrails/waymarkedtrails-shields/blob/35642d5609572f4824d03b7e51d20a599ffc77bc/wmt_shields/styles/osmc_symbol.py#L272

For some symbols we had the plain version without direction and a "left" version, so I added the "right" version for avoid potential confusion. I don't think it needs to be "advertised" explicitly, but it would be good to have it supported for completeness.

lonvia reviewed (approved):
Ha, looks like I didn't pay attention then.

https://github.com/waymarkedtrails/waymarkedtrails-website/blob/master/src/OsmcApp.svelte#L16 could also do with an appropriate update.
